less/navbar.less

```diff
@@ -482,6 +480,10 @@
       }
     }
   }
+
+  .badge {
+    background-color: @brand-red;
```

Comment:
Let's just make all badges red in badges.less

Reply:
Ok, will do that. Didn't want to go that far, so I made it very specific.